grpc/src/main/java/com/linecorp/armeria/server/grpc/FramedGrpcService.java

```diff
@@ -236,10 +236,11 @@ protected HttpResponse doPost(ServiceRequestContext ctx, HttpRequest req) throws
                 } catch (IllegalArgumentException e) {
                     final Metadata metadata = new Metadata();
                     final GrpcExceptionHandlerFunction exceptionHandler = registry.getExceptionHandler(method);
+                    final Status status = Status.UNKNOWN.withCause(e);
```

Comment:
`IllegalArgumentException` is raised when an invalid timeout value is set. So what do you think of using `INVALID_ARGUMENT`?

Reply:
Currently, we convert `IllegalArgumentException` to `UNKNOWN` in `DefaultGrpcExceptionHandlerFunction` so I followed that but we could probably use INVALID_ARGUMENT.

Reply:
In `DefaultGrpcExceptionHandlerFunction`, we don't know why `IllegalArgumentException` is raised so `UNKNOWN` is probably a good choice.

However, the reason for `IllegalArgumentException` is clear. This is an error that occurred internally in Armeria, so we can appropriately convert it to a status.

Reply:
Yeah, that makes sense. 👍